research/deeplab/datasets/.gitignore

```diff
@@ -0,0 +1,2 @@
+# datafiles
```

Comment:
please remove this file.

Reply:
Removed :-) thought it would help contribute.